quinn-proto/src/connection/mod.rs

```diff
@@ -233,6 +233,10 @@ pub struct Connection {
     stats: ConnectionStats,
     /// QUIC version used for the connection.
     version: u32,
+
+    /// qlog streamer to ouput events as JSON text sequences
+    #[cfg(feature = "qlog")]
+    qlog_streamer: Option<qlog::streamer::QlogStreamer>,
```

Comment:
Nit: prefer to import the `QlogStreamer` (especially since it has a crate-specific name anyway), with a conditional import at the top of the module.